v1/src/main/java/com/google/cloud/teleport/spanner/ddl/Ddl.java

```diff
@@ -367,7 +367,7 @@ public String createProtoBundleStatement() {
       appendable.append("CREATE PROTO BUNDLE (");
       for (String protoTypeFqn : protoBundle()) {
         appendable.append("\n\t");
-        appendable.append(protoTypeFqn);
+        appendable.append("`" + protoTypeFqn + "`");
```

Comment:
Please get the quote string from `NameUtils.literalQuote` and use it here.
For example line number 388 in this file.

Reply:
fixed it.